Sources/SwiftCompilerPluginMessageHandling/PluginMessages.swift

```diff
@@ -85,12 +85,13 @@ internal enum PluginToHostMessage: Codable {
 
   enum MacroRole: String, Codable {
     case expression
-    case freeStandingDeclaration
+    case declaration
```

Comment:
I'm renaming this because macro roles typically don't include  "freestanding" vs "attached". Does this need a corresponding change on the compiler side? If so I can revert this. @rintaro

Reply:
It does not _need_ a corresponding change because I believe `freeStandingDeclaration` and `expression` aren't actually used in `expandFreestandingMacro` message. But yes, we should sync this file with swift repo one.
But you/we can do it later (after you merge this PR)